src/configure

```diff
@@ -1395,7 +1395,7 @@ Optional Features:
                           Enable OpenMP for fast formats (usually a bad idea -
                           use fork instead)
   --enable-mpi            Use OpenMPI
-  --enable-rexgen         Enable use of rexgen library (experimental)
+  --enable-rexgen         Enable use of rexgen library (experimental!)
```

Comment:
Maybe replace `experimental!` with `unreliable`. This isn't really an experiment for some upcoming feature - it's a feature that's been around for a long time and never became reliable and isn't expected to become reliable now.